Include/internal/pycore_bitutils.h

```diff
@@ -140,6 +141,45 @@ _Py_popcount32(uint32_t x)
 }
 
 
+static inline int
+_Py_popcount64(uint64_t x)
+{
+#if (defined(__clang__) || defined(__GNUC__))
+
+#if SIZEOF_INT >= 8
+    Py_BUILD_ASSERT(sizeof(x) <= sizeof(unsigned int));
+    return __builtin_popcount(x);
+#else
+    // The C standard guarantees that unsigned long will always be big enough
+    // to hold a uint32_t value without losing information.
+    Py_BUILD_ASSERT(sizeof(x) <= sizeof(unsigned long));
+    return __builtin_popcountl(x);
+#endif
```

Comment:
Why not just always call `__builtin_popcountl(x)` without the #if conditional?  No platform I've ever seen that we support has an `int` who's size is > 32-bits.

Reply:
I reverted my changes for Windows for now. I may propose a follow-up PR for Windows.